lib/smart_proxy_remote_execution_ssh/webrick_ext.rb

```diff
@@ -0,0 +1,11 @@
+module SmartProxyRemoteExecutionSsh
+  module WEBrickExt
+    # An extension to ::WEBrick::HTTPRequest to expost the socket object for highjacking for cockpit
```

Comment:
@mvollmer this set of changes should make the code compatible with webrick as well (at least during my tests everything worked)

Reply:
I get this error with a prdocution setup on Centos 7:

```
/usr/share/gems/gems/smart_proxy_remote_execution_ssh-0.2.0/lib/smart_proxy_remote_execution_ssh/webrick_ext.rb:10:in `<module:SmartProxyRemoteExecutionSsh>': private method `prepend' called for WEBrick::HTTPRequest:Class (NoMethodError)
```

Reply:
I've pushed a new commit that should fix that